docs/SETUP_REMOTE.md

```diff
@@ -29,6 +29,14 @@ There are other options like a Docker container per person though.
 
 Install tools you need to perform remote processing.
 
+If some tool require shell initialization scripts or special environment variables you should add them to your `~/.bashrc` or `~/.bash_profile` file depending on your remote operating system.
+
+Be sure to add all your configurations at the beginning of the file before this line :
+
+  ```bash
+  # If not running interactively, don't do anything
```

Comment:
>depending on your remote operating system

Already mentioned

>Be sure to add all your configurations at the beginning of the file before this line

Does not prevent you from adding setup you need if you don't have such line in the file, so I guess it's ok.

Also, Arch, Gentoo, etc users are usually pro-enough to find out solution on their own :)